src/screens/AuthScreen/AuthScreen.tsx

```diff
@@ -11,18 +12,46 @@ import { getUserData } from './Util';
 const AuthScreen = () => {
   const { setLoggedInUserData } = useContext(AuthContext);
   const [githubView, setGithubView] = useState(false);
+  const [addressbarURL,setAdressbarURL] = useState<String>("")
+  const [loading,setLoading] = useState(false)
+  const [key,setKey] = useState(1)
 
   const handleSignIn = () => {
     setGithubView(true);
   };
-
+  
   if (githubView) {
     return (
       <ScrollView contentContainerStyle={AuthViewStyle.container}>
+        <View style={AuthViewStyle.addressBarStyle}>
+          {loading ?
+            <ActivityIndicator style={{marginLeft: 5}} size={25} color="#fff" /> :
+            <TouchableOpacity 
+              onPress={() => setGithubView(false)}
+            >
+              <Text style={AuthViewStyle.addressBarCancel}>Cancel</Text>
+            </TouchableOpacity>
+          }
+          <Text style={AuthViewStyle.addressBarLink}>{addressbarURL}</Text>
+          {loading ? 
+            null : 
+            <TouchableOpacity
+              onPress={() => setKey(key + 1)}
+            >
+              <Image 
+                source={Images.refreshIcon}
+                style={AuthViewStyle.addressBarIcon}
+              />
+            </TouchableOpacity>
+          }
+          
+        </View>
         <WebView
+          key={key}
```

Comment:
why did you use key here?

Reply:
webview doesn't have a reload function, so it updates the page if a state updates, so the key updates when we click on the refresh button and then the page reloads